CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+Code of Conduct
+===============
+
+The following text is derived from the [Django Project's Code of
+Conduct](https://www.djangoproject.com/conduct/), which is licensed as
+[Creative Commons Attribution](https://creativecommons.org/licenses/by/3.0/).
+
+Like the technical community as a whole, the restic team and community is made
+up of a mixture of professionals and volunteers from all over the world,
+working on every aspect of the mission - including mentorship, teaching, and
+connecting people.
+
+Diversity is one of our huge strengths, but it can also lead to communication
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors and those
+seeking help and guidance.
+
+This isn't an exhaustive list of things that you can't do. Rather, take it in
+the spirit in which it's intended - a guide to make it easier to enrich all of
+us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the restic project. This
+includes IRC, the issue tracker, the forum, and any other forums created by the
+project team which the community uses for communication. In addition,
+violations of this code outside these spaces may affect a person's ability to
+participate within them.
```

Comment:
As a general principle, I don't like the idea that the code of conduct for one project would enforce behaviour in other forums where the code of conduct might be different. It's somewhat like one country defining rules of conduct in other countries. This is confusing. I expect this is to cover extreme cases such as when somebody has done something so troublesome as to make it a concern that their conduct in another forum might in any way reflect poorly on this project.

I think there is a wide range of possible concerns here, and the starting point should be to leave it out entirely. However, if you want to cover this point - perhaps being more explicit would help avoid this being a concern: "Although it is not the responsibility of the restic project to define and enforce code of conduct outside of the restic project, there may be situations that require the restic project to distance itself from the actions of a contributor. In this unfortunate situation, a contributor may be asked to leave the community so as to preserve the good reputation of the project."

Reply:
> As a general principle, I don't like the idea that the code of conduct for one project would enforce behaviour in other forums where the code of conduct might be different. It's somewhat like one country defining rules of conduct in other countries. 

You mean like the GDPR?  ha...ha...

> I think there is a wide range of possible concerns here, and the starting point should be to leave it out entirely. However, if you want to cover this point - perhaps being more explicit would help avoid this being a concern...

I appreciate your trying to be reasonable, but it's still a very bad idea, no matter how nicely or carefully you word it.  It's just a ticking time bomb.

This is backup software.  We just want good software.  That should be our only goal here.  Politics and societal issues ought to be explicitly barred from being relevant to the project and participation in it.